recipe/meta.yaml

```
@@ -41,15 +42,15 @@ test:
     - pytest -vv --pyargs ipywidgets 
 
 about:
-  home: https://github.com/ipython/ipywidgets
+  home: https://github.com/jupyter-widgets/ipywidgets
```

```suggestion
  home: https://jupyter.org
```
https://github.com/jupyter-widgets/ipywidgets/blob/8.1.2/python/ipywidgets/setup.cfg#L10